packages/codegen/generate.hooks.ts

```diff
@@ -379,7 +380,11 @@ export function generateReactHooks<
     }
 
     function useAllObjects (variables: QueryVariables, options?: QueryHookOptions<IUseObjectsQueryReturnType<GQLObject>, QueryVariables>) {
-        const { objs, count, error, loading, refetch: _refetch, fetchMore, stopPolling } = useObjects(variables, options)
+        const fetchPolicy = get(options, 'fetchPolicy')
+        const { objs, count, error, loading, refetch: _refetch, fetchMore, stopPolling } = useObjects(variables, {
+            ...options,
+            fetchPolicy: fetchPolicy && fetchPolicy !== 'no-cache' ? fetchPolicy : 'network-only',
```

Comment:
Network only is default policy?

Reply:
Oh, I miss description and doc link...

Reply:
Anyway I can't understand why you compare user input of fetchPolicy with `no-cache` value?

Reply:
We use no-cache by default, but "fetchMore" now works differently with "no-cache". So to restore functionality of this hook, we had to make it "network-only".
To achieve the expected behavior of the hook, I need to make sure that "no-cache" does not get here